src/bin/console/console.cpp

```diff
@@ -92,6 +97,26 @@ std::string remove_coloring(const std::string& input, bool remove_rl_codes_only
   std::regex expression{"(" + sanitized_sequences + ")"};
   return std::regex_replace(input, expression, "");
 }
+
+void find_all_join_graphs(const std::shared_ptr<AbstractLQPNode>& lqp, std::vector<JoinGraph>& join_graphs,
```

Comment:
Does it "find" join graphs that are already stored somewhere or does it create/gather them?